### ISSUE TITLE
More token enhancements

### ISSUE THREAD

**Issue by ThePants999** [ThePants999/tts-comp-ftc-base-lua-scripts#19]
Currently waiting on Trashpanda for graphics for:

- +1S / +1T / +1A (minus equivalents wouldn't see much use but if they're just a recolour then why not)
- Generic "+ movement" / "- movement", no numbers needed IMO as they'll relate to fairly unique effects
- Generic "state changed", as a decent option for something like SM "put a unit in X doctrine", DW "put a unit in X chapter tactic", Sisters "give a unit an extra sacred rite" sort of stuff 
- revamp of 1/2/3 to adv/charge/Ld (if we were doing a full refresh for graphical consistency purposes, it wouldn't be a big deal to lose +/- 3)
- revamp of action in progress/completed (though would be nice if they were better)

Also would be nice to have +1/-1 damage

**Comment by ThePants999** [comment]
Exploding 6s

**Comment by ThePants999** [comment]
Most of this is now sorted in v2.47. +1 advance has the wrong icon, and still looking to add +1/-1 damage and exploding 6s.
